### PR TITLE
Responsive Navigation Bar Added.

### PR DESCRIPTION
**Pull request by sanjay270899**
#### Issue Number
ISSUE #12 

#### Describe the changes you've made
Created a responsive Header with a sticky position at the top.

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA

#### Additional context
Screenshots about the feature.
image
For mobile.
image

#### Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **[CONTRIBUTING](https://github.com/GameofSource-GFG/Web-Development/blob/main/CONTRIBUTING.md)** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.
- [ ] Participating in SWOC2021.
- [x] Participating in DWOC.

### CODE REVIEW

**Comment by sanjay270899** [comment]
Do add DWOC and Level label before merging. Thank You.

File: client/src/components/Header/index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                    Contact Us
+                </div>
+                <div className="mobile-nav-links">
+                    Log In
+                </div>
+                <div className="mobile-nav-links">
+                    Sign Up
+                </div>
+            </nav>
+        </>
```
Comment (07souravkunda) [inline client/src/components/Header/index.js:51]: Create a list of objects having link name and link path and map through it to render navs

**Comment by 07souravkunda** [comment]
@sanjay270899 don't use node-sass. Only use test.module.css files for styling.

**Comment by sanjay270899** [comment]
What do you mean by test.moudel.css files? Please elaborate, or refer me to any article.

**Comment by 07souravkunda** [comment]
Refer this article https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/

File: client/src/components/Header/index.js (hunk 1 above)
Comment (sanjay270899) [reply client/src/components/Header/index.js]: Done. I have changed the styling from scss to module.css.
Also, I have added a list of link names and paths.

**Comment by 07souravkunda** [comment]
@sanjay270899 good work. Just one additional change make nav-link active when respective route is selected.

**Comment by sanjay270899** [comment]
> @sanjay270899 good work. Just one additional change make nav-link active when respective route is selected.

How should I implement that? Means how will I know which nav-link is active?

**Comment by 07souravkunda** [comment]
https://stackoverflow.com/questions/47879663/root-navlink-always-get-active-class-react-router-dom
Refer this. Hope it helps you

**Comment by sanjay270899** [comment]
For this, we have to install a dependency. Right?

**Comment by 07souravkunda** [comment]
You have installed react router already so no need

**Comment by sanjay270899** [comment]
Done.  Do add Labels before merging.

Thanks for the suggestions.

**Comment by 07souravkunda** [comment]
@sanjay270899 I think your pr is ready to be merged.
SWOC evaluation will start after 4th jan. Do you want me to merge it now?

**Comment by sanjay270899** [comment]
@07souravkunda, I haven't registered for SWOC. So you can merge this PR today.

**Comment by 07souravkunda** [comment]
ok. Resolve the conflicts. I will merge it.

**Review by 07souravkunda — APPROVED** [review]

**Comment by sanjay270899** [comment]
@07souravkunda Resolved.

**Comment by 07souravkunda** [comment]
Merged. Start working on #14

**Pull request merged**